data/harisdev.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Computer Science guy in a Software Engineering world!",
+    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png"
```

Comment:
The `avatar` field is not required. Your avatar is automatically fetched from your GitHub profile.
```suggestion
```

Reply:
Done